### PR TITLE
Hotfix 2.8.4

### PR DESCRIPTION
**Pull request by amilan**
Hotfix to add the fixes to the broken fscan macro (#1218, #1220 )

### CODE REVIEW

**Comment by amilan** [comment]
Hi,

This is a PR to create a new hotfix release including the fixes for the broken fscan macro.

It is in WIP because I cannot make the `makeman` script from taurus work to update the versions of the documentation pages.

Any help/feedback will be welcome.

**Comment by reszelaz** [comment]
Thanks for working on the hot fix! 
Could you please post here the error from makeman? Some ideas of what may go wrong: Do you have help2man installed? Have you seen that there is a known issue with generating man for Spock?

**Comment by amilan** [comment]
Hi,
Thanks for the ideas, unfortunately I have to answer yes to both of them.
I'm running the script from the doc folder and the output that I got is the following:

```shell
Installing diffractometeralignment script to /tmp/tmp.XtTvSFtQSs/bin
rm: cannot remove ‘/projects/tmp/sardana/sardana/doc/*.1’: No such file or directory
/tmp/tmp.XtTvSFtQSs/bin
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/diffractometeralignment.1 --version-string= ./diffractometeralignment
help2man: can't get `--help' info from ./diffractometeralignment
Try `--no-discard-stderr' if option outputs to stderr
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/hklscan.1 --version-string= ./hklscan
help2man: can't get `--help' info from ./hklscan
Try `--no-discard-stderr' if option outputs to stderr
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/macroexecutor.1 --version-string= ./macroexecutor
help2man: can't get `--help' info from ./macroexecutor
Try `--no-discard-stderr' if option outputs to stderr
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/MacroServer.1 --version-string= ./MacroServer
help2man: can't get `--help' info from ./MacroServer
Try `--no-discard-stderr' if option outputs to stderr
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/Pool.1 --version-string= ./Pool
help2man: can't get `--help' info from ./Pool
Try `--no-discard-stderr' if option outputs to stderr
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/Sardana.1 --version-string= ./Sardana
help2man: can't get `--help' info from ./Sardana
Try `--no-discard-stderr' if option outputs to stderr
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/sardanatestsuite.1 --version-string= ./sardanatestsuite
help2man: can't get `--help' info from ./sardanatestsuite
Try `--no-discard-stderr' if option outputs to stderr
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/sequencer.1 --version-string= ./sequencer
help2man: can't get `--help' info from ./sequencer
Try `--no-discard-stderr' if option outputs to stderr
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/spock.1 --version-string= ./spock
help2man: can't get `--help' info from ./spock
Try `--no-discard-stderr' if option outputs to stderr
--------------------
help2man -N -o /projects/tmp/sardana/sardana/doc/man/ubmatrix.1 --version-string= ./ubmatrix
help2man: can't get `--help' info from ./ubmatrix
Try `--no-discard-stderr' if option outputs to stderr
```

(I removed the part of moving and installing things from this output)

**Comment by reszelaz** [comment]
Which version of help2man do you use? I use: GNU help2man 1.47.4.

But I would not worry too much about it. To unblock you I've recreated the manpages and pushed them to your branch.

Actually I think that these man pages do not add too much value and make the release and hotfix process more complicated. Personally I'm more used to see the help with `--help`.
This may come from the way how Debian packages are created, maybe @cpascual could advice whether we could get rid of them in the upstream release process?

**Comment by amilan** [comment]
Thanks a lot @reszelaz!

I'm using help2man 1.47.8 in CentOS and I also tried with the version 1.47.11 in macOS. Both with the same results.

Anyway, thanks for generating the man pages.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,16 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 This file follows the formats and conventions from [keepachangelog.com]
 
-## [Unreleased]
+## [2.8.4] 2019-11-06
+
+### Fixed
```
Comment (reszelaz) [inline CHANGELOG.md]: Here we miss the bug fix propsed by @tiagocoutinho

**Comment by reszelaz** [comment]
I just realized that the hotfix process was not finished. Since this was not a normal hotfix but more a backport of a fix from the develop branch I had to proceed differently finish with the hotfix process. I just manually added 2.8.4. release to the CHANGELOG.md and now the process is completed.

**Pull request merged**